src/modules/booking/entities/booking.entity.ts

```diff
@@ -1,38 +1,50 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { AutoMap } from '@automapper/classes';
 import { BaseEntity } from 'src/common/entity/base.entity';
 import { Field } from 'src/modules/field/entities/field.entity';
+import { User } from 'src/modules/user/entities/user.entity';
 import { Column, Entity, JoinColumn, ManyToOne } from 'typeorm';
 
 export enum BookingStatus {

```

Comment:
uppercase for enum

export enum BookingStatus {
  DISABLED = 'disabled',
  REJECTED = 'rejected',
 ...
}